docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -43,28 +45,74 @@ Proceed to the <a href="openvino_docs_install_guides_installing_openvino_linux.h
 
 @sphinxdirective
 
-.. _gpu guide windows:
+.. tab:: Linux
 
-@endsphinxdirective
+   After installing OpenVINO, follow the steps below to be able to work on GPU:
+
+   1. Go to the `install_dependencies` directory:
+
+
+   .. code-block:: sh
+
+      cd <INSTALL_DIR>/install_dependencies/
+
+   2. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. The drivers are not included in the package. To install, run this script:
+
+   .. code-block:: sh
+
+      sudo -E ./install_NEO_OCL_driver.sh
+
+   .. note:: To use the **Intel® Iris® Xe MAX Graphics**, see the `Intel® Iris® Xe MAX Graphics with Linux`_ page for driver installation instructions.
+
+      .. _Intel® Iris® Xe MAX Graphics with Linux: https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html
+   
+   The script compares the driver version on the system to the current version. If the driver version on the system is higher or equal to the current version, the script does not install a new driver. If the version of the driver is lower than the current version, the script uninstalls the lower version and installs the current version with your permission:
+
+   .. image:: ../img/NEO_check_agreement.png
+
+   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output: 
```

Comment:
```suggestion
   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command-line output: 
```